spec/hash_map/mapper_spec.rb

```diff
@@ -17,17 +17,21 @@ module HashMap
         phone: nil
       }
     end
-    let(:data_structure) do
-      [
-        { key: [:first_name], from: [:name] },
-        { key: [:last_name], proc: proc { |input| "#{input[:first_surname]} #{input[:second_surname]}" } },
-        { key: [:language], from: [:address, :country, :language], transform: proc {|context, value| value.downcase } },
-        { key: [:email, :address], from: [:email] },
-        { key: [:email, :type], default: :work },
-      ]
+
+    class HashMap
```

Comment:
Can we called with different name. I do not want to override HashMap in the tests